db/migrate/20170203141631_add_maintenance_mode_to_storages.rb

```diff
@@ -0,0 +1,5 @@
+class AddMaintenanceModeToStorages < ActiveRecord::Migration[5.0]
+  def change
+    add_column :storages, :maintenance_mode, :boolean
```

Comment:
Foun it...https://github.com/ManageIQ/manageiq-providers-vmware/pull/6/files#diff-2ab3234a0059931160bdf4d4eb6152c5R285

Seems the column is a boolean named `maintenance`, so let's change here for consistency